src/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import Routes from './routes';
+import GlobalStyle from './styles/GlobalStyle';
+
+export default function App() {
+	return (
+		<Router>
+			<Routes />
+			<GlobalStyle />
```

Comment:
Place the `GlobalStyle` for first, because right now styles are rendering in a wrong order, see:

![CeEV3NfWqxpvN1ADohxPGwNe](https://user-images.githubusercontent.com/4838076/78775149-a55e6d00-798d-11ea-836d-0b5b0fd87e30.png)

Because styled-component will render the components and only when all this job is done, it will add the global style. But the global one must be the first processed style, then you can overwrite the global style on each component if it is needed of course.